src/app/pages/details/common/hooks.tsx

```diff
@@ -3,28 +3,26 @@ import tableOfContentsHtml from '~/models/table-of-contents-html';
 import partnerFeaturePromise, {tooltipText} from '~/models/salesforce-partners';
 import shuffle from 'lodash/shuffle';
 import {camelCaseKeys} from '~/helpers/page-data-utils';
+import useDetailsContext from '../context';
 
-export function useTableOfContents(model) {
-    const webviewLink = model.webviewRexLink || model.webviewLink;
-    const isRex = Boolean(model.webviewRexLink);
-    const [tocHtml, setTocHtml] = useState('');
+export function useTableOfContents() {
+    const model = useDetailsContext();
+    const webviewLink = model.webviewRexLink;
+    const [tocHtml, setTocHtml] = useState<string | void>('');
```

Comment:
If you return an empty string from the err function inside the `then` inside `cnxFetch` return, you won't need the void type

Reply:
Thanks!